enhancements/ota/distribute-secondary-metadata-as-container-image.md

```diff
@@ -0,0 +1,165 @@
+---
+title: cincinnati-graph-data-package-distribute-and-consume-in-a-container-image-repository
+authors:
+  - "@sjunker"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-05-07
+last-updated: 2020-05-07
+status: provision
+see-also:
+-
+replaces: []
+superseded-by: []
```

Comment:
nit: I think you can just drop properties that have no content; we can always add them back if we grow some content to put inside.

Reply:
I left these there so I don't need to look up what the possible options are ;-)

Reply:
Please mark this as resolved.